### DISCUSSION TITLE
有办法为配置Key默认添加文件名前缀么？

### DISCUSSION THREAD

**Discussion by zhangzw218** [apolloconfig/apollo#3615] [Q&A] [answered]
我发现apollo对properties支持比较好，所以希望都用properties来配置属性。
而我原本的配置是
CacheConfig.Json

```
{
  "CacheConfig":{
    "EnableTitanCache":true,
    "CacheExpireMinute":20,
    "CacheKeyPrefix":"entity_cache:",
  }
}
```
现在改成了
CacheConfig.properties

```
CacheConfig:EnableTitanCache=true
CacheConfig:CacheExpireMinute=20
CacheConfig:CacheKeyPrefix="entity_cache"
```
可以看到上面有共同的前缀CacheConfig，我在想，可以在apollo中配置的时候，只写EnableTitanCache，在Net中加载配置的时候，自动加上文件名前缀么？
比如下面这样:

CacheConfig.properties

```
EnableTitanCache=true
CacheExpireMinute=20
CacheKeyPrefix="entity_cache"
```
但是在代码使用中，可以达到和上面一样的效果？
比如NetCore中这样获取：
`Services.AddOptions<CacheOptions>().Bind(configuration.GetSection("CacheConfig"));`

**Comment by nobodyiam** [comment, marked as answer]
这个规则比较定制了，建议可以在客户端侧封装一层来实现这个逻辑
